src/test/kotlin/pt/up/fe/ni/website/backend/controller/AuthControllerTest.kt

```diff
@@ -0,0 +1,185 @@
+package pt.up.fe.ni.website.backend.controller
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.hamcrest.Matchers.startsWith
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.security.crypto.password.PasswordEncoder
+import org.springframework.test.annotation.DirtiesContext
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.get
+import org.springframework.test.web.servlet.post
+import pt.up.fe.ni.website.backend.model.Account
+import pt.up.fe.ni.website.backend.model.CustomWebsite
+import pt.up.fe.ni.website.backend.repository.AccountRepository
+import pt.up.fe.ni.website.backend.utils.TestUtils
+import java.util.Calendar
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@AutoConfigureTestDatabase
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+class AuthControllerTest @Autowired constructor(
+    val repository: AccountRepository,
+    val mockMvc: MockMvc,
+    val objectMapper: ObjectMapper,
+    passwordEncoder: PasswordEncoder
+) {
+    final val testPassword = "testPassword"
+
+    // TODO: Make sure to add "MEMBER" role to the account
```

Comment:
This can be referenced in the issue mentioned above

Reply:
Sure